src/priors/matern.jl

```diff
@@ -56,31 +57,22 @@ function to_1d_sde(p::Matern)
     L_breve = zeros(q + 1)
     L_breve[end] = 1.0
 
-    return LTISDE(F_breve, L_breve)
-end
-function to_sde(p::Matern)
     d = p.wiener_process_dimension
-    _sde = to_1d_sde(p)
-    F_breve, L_breve = drift(_sde), dispersion(_sde)
-    F = kron(I(d), F_breve)
-    L = kron(I(d), L_breve)
+    F = IsometricKroneckerProduct(d, F_breve)
+    L = IsometricKroneckerProduct(d, L_breve)
     return LTISDE(F, L)
 end
 function discretize(p::Matern, dt::Real)
```

Comment:
Whie we're at it, check that make_transition_matrices does the correct thing as well.